model/resource/browser.yaml

```diff
@@ -54,3 +54,19 @@ groups:
           to retrieve brands and platform individually from the User-Agent Client Hints API.
           To retrieve the value, the legacy `navigator.userAgent` API can be used.
         examples: ['Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36']
+      - id: page.url
```

Comment:
Is `page.*` really an attribute of the **resource** `browser`? Shouldn't that be rather a signal-level attribute?

In particular: if a user navigates to a different page would that also imply that there's a new instance of the browser SDK? If not, the `page.*` must not be a resource attribute, because resource attributes must not change during the lifetime of a resource / SDK instance. 


Apart from the above, can we use `browser.page.url.full` instead? To be consistent with the `url.*` namespace. Once we have a solution for https://github.com/open-telemetry/semantic-conventions/issues/339, we will be able to simply replace it with a reuse of the `url.full` attribute under the `browser.page.*` namespace

Reply:
@AlexanderWert yes, in my intended usage of these attributes they are not at the resource level - I added them here so that all attributes under `browser` namespace are at one place. I realize this one place is in registry, which didn't exist when this namespace was introduced. I can create the registry entry for browser now, but have one question - these attribute are intended to be added by sdk level processors into both spans and events. Do you want me to create both model/trace/browser.yaml and model/logs/browser.yaml, duplicated?

cc: @martinkuba

Reply:
@scheler FYI I have opened a PR with `browser` moved to the registry. I would like in the future to change your PR to add new attributes to the registry directly